gef-ui/views/templates/name-application.njk

```diff
@@ -0,0 +1,32 @@
+{% extends "index.njk" %}
+{% import '_macros/success-message.njk' as success %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block pageTitle %}Confirm eligiblity (mandatory criteria){% endblock %}
+
+{% block content %}
+
+  {% if successMessage %}
+    {{ success.message(
+      message = successMessage
+    )}}
+  {% endif %}
+    <h1 class="govuk-heading-l">
+      Name application
+    </h1>
+
+    {{ criteria | safe }}
+
+    <form action="name-application" method="post">
```

Comment:
I don't think `action="name-application"` does anything? as the server would pick up the form via the post method/route